src/app/pages/ArticlePage/ArticlePage.styles.ts

```diff
@@ -93,18 +93,10 @@ export default {
   topStoriesSection: ({ spacings, mq }: Theme) =>
     css({
       marginBottom: `${spacings.TRIPLE}rem`,
+
       [mq.GROUP_4_MIN_WIDTH]: {
-        display: 'block',
         marginBottom: `${spacings.FULL}rem`,
         padding: `${spacings.DOUBLE}rem`,
       },
```

Comment:
I've removed the experiment-specific styling added to `topStoriesSection` here, but this was originally combined with `featuresSection` as they have the same styles ([commit showing this](https://github.com/bbc/simorgh/commit/bdd3edd1d6abde514f7e90ff6af4fda0d8426026)). 
Should I combine these together again or keep them separate? We don't have any plans at the moment with either section, but just curious if there was any preference to keep these separate. Happy either way.